lms/views/dashboard/api/course.py

```diff
@@ -22,16 +63,44 @@ def __init__(self, request) -> None:
         self.h_api = request.find_service(HAPI)
         self.organization_service = request.find_service(OrganizationService)
 
+    @view_config(
+        route_name="api.dashboard.courses",
+        request_method="GET",
+        renderer="json",
+        permission=Permissions.DASHBOARD_VIEW,
+        schema=ListCoursesSchema,
+    )
+    def courses(self) -> APICourses:
+        courses = self.course_service.get_courses(
+            h_userid=self.request.user.h_userid if self.request.user else None,
+        )
+
+        limit = min(PAGINATION_LIMIT, self.request.parsed_params["limit"])
+        if cursor := self.request.parsed_params.get("cursor"):
+            cursor_course_name, cursor_course_id = _get_cursor_values(cursor)
+            courses = courses.filter(
+                (Course.lms_name, Course.id) > (cursor_course_name, cursor_course_id)
+            )
+
+        courses = courses.limit(limit).all()
+        return {
+            "courses": [
+                APICourse(id=course.id, title=course.lms_name) for course in courses
+            ],
+            "next": _next_parameter(courses),
```

Comment:
`next` here at the top level result, should we include it on a "pagination" object or similar?